app/main/views/audits.py

```diff
@@ -115,6 +115,27 @@ def list_audits():
     elif object_id:
         abort(400, 'object-id cannot be provided without object-type')
 
+    after_event_id = request.args.get('after-event-id')
```

Comment:
Oh yes so I thought about this: here we go...

I figured `earliest_for_each_object` was more a paging-y type thing so fitted in with `per_page` et al while `after-audit-id` was more of a filtering operation and so fitted better with `object-id`.

The mental gymnastics required for that rationalization...